README.md

```diff
@@ -61,6 +60,115 @@ yarn start
 
 Finally, we can navigate a browser window to `localhost:8080` to view generated data.
 
+### Running with Docker
+
+Optionally, you can build and run sourcecred in a container to avoid installing dependencies
+on your host. First, build the container:
+
+```bash
+$ docker build -t sourcecred .
+```
+
+If you want to build and customize the `SOURCECRED_DIRECTORY`, you can
+set that as a `--build-arg`:
+
+```bash
+$ docker build --build-arg SOURCECRED_DEFAULT_DIRECTORY=/tmp/data -t sourcecred .
+```
+
+If you don't want to build, the container is also provided at [vanessa/sourcecred](https://hub.docker.com/r/vanessa/sourcecred)
+
+```bash
+$ docker pull vanessa/sourcecred
```

Comment:
I'm uncomfortable referencing a "third party" docker file in the README, especially since SourceCred changes pretty frequently, so unless we do a good job coordinating, the docker file would go out of date.

what's the workstream involved in keeping an official SourceCred docker file up to date? can we have a script like `scripts/update_dockerfile.sh` that we can invoke in [deploy.sh](https://github.com/sourcecred/sourcecred/blob/master/scripts/deploy.sh) and use to ensure that we update our dockerfile on every release?

Reply:
I'd be happy to remove this (and just pushed a commit to do that)! The only thing that we'd have to worry about keeping up to date is an actual provided container, the typical use case without that would be for the user to clone and build, at which point they build the latest from the code. If you are interested in providing an actual image (you could grab sourcecred on Docker Hub) I'd be glad to do another PR with a build -> deploy script (Travis, Circle, now GitHub CI, etc, choose your preference).

Reply:
Yeah, I think maintaining an official docker container makes sense; especially if you'll provide the build/deply script. I just grabbed the `sourcecred` on Docker Hub. Let's go with CircleCI integration since we already use that for CI.

Reply:
It’s a plan! If it’s ok with you, I’ll do that in a separate PR to keep each well scoped and smaller in size.

Reply:
sgtm

Reply:
Great! If you want to create an issue an assign me, that works, otherwise I'll just remember and add to my TODO. Resolving this discussion.